features/steps/steps.go

```diff
@@ -59,6 +68,15 @@ func (c *Component) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^the following Export Start events are produced:$`,
 		c.theFollowingExportStartEventsAreProduced,
 	)
+	ctx.Step(`^the following recipe is used to create a dataset based on the given cantabular dataset:$`,
```

Comment:
Is this the right way to be testing this? It's quite a large layer of abstraction in the test code.. Wouldn't it be better to explicitly be laying out what the service actually interacts with? I.E. the available dataset, metadata, dimensions etc that this step is comprised of?

Reply:
looking at it both ways might give the answer

Reply:
I have added a get-dataset-json-alternate.feature to show the 2 different approaches. The latter is a lot more verbose but see what you think